src/UI/controllers/HTMLTooltipUIController.js

```diff
@@ -150,6 +150,7 @@ export class HTMLTooltipUIController extends UIController {
                 },
                 onManage: (type) => {
                     this._onManage(type);
+                    this.removeTooltip();
```

Comment:
I think we can do this in general for both Mac/Windows, I don't see anything breaking on Mac yet.

Reply:
Also, IMO it's better to have the tooltip remove logic outside of `_onManage`